Cython/Includes/numpy/math.pxd

```diff
@@ -1,133 +0,0 @@
-# NumPy math library
```

Comment:
FWIW tried something like this locally for `numpy.math`

```suggestion
cdef extern from * nogil:
    """
    #error `numpy.math` is removed. Please use `libc.math` instead.
    """
```

Using a simple module like this one

```cython
# filename: mymod.pyx

from numpy.math cimport *
```

Tried building it and got the expected error

```
% cythonize -i mymod.pyx
Compiling /Users/jkirkham/cython_test/mymod.pyx because it changed.
[1/1] Cythonizing /Users/jkirkham/cython_test/mymod.pyx
/Users/jkirkham/miniforge/envs/cython/lib/python3.10/site-packages/Cython/Compiler/Main.py:381: FutureWarning: Cython directive 'language_level' not set, using '3str' for now (Py3). This has changed from earlier releases! File: /Users/jkirkham/cython_test/mymod.pyx
  tree = Parsing.p_module(s, pxd, full_module_name)
/Users/jkirkham/cython_test/mymod.c:1205:6: error: `numpy.math` is removed. Please use `libc.math` instead.
    #error `numpy.math` is removed. Please use `libc.math` instead.
     ^
1 error generated.
error: command '/Users/jkirkham/miniforge/envs/cython/bin/arm64-apple-darwin20.0.0-clang' failed with exit code 1
```

Maybe this is an option for consideration?